job-simulation.rb

```diff
@@ -1,4 +1,67 @@
 require './Stack.rb'
 require './Queue.rb'
 
+waiting = Queue.new
+workers = Stack.new
 
+puts "How many people do you have? (Must be greater than 6)"
+initial_num = gets.chomp.to_i
+
+1.upto(initial_num) do |n|
+  waiting.enqueue(rand(100..999))
+end
+
+puts "\nInitial Waiting List #{waiting.store}"
+
+new_hires = waiting.dequeue(6)
```

Comment:
dequeue should not take an argument. It should remove exactly one element each time dequeue is called.